README.md

```diff
@@ -1,11 +1,11 @@
 # Onvif Camera Device Service
 [![Build Status](https://jenkins.edgexfoundry.org/view/EdgeX%20Foundry%20Project/job/edgexfoundry/job/device-onvif-camera/job/main/badge/icon)](https://jenkins.edgexfoundry.org/view/EdgeX%20Foundry%20Project/job/edgexfoundry/job/device-onvif-camera/job/main/) [![Code Coverage](https://codecov.io/gh/edgexfoundry/device-onvif-camera/branch/main/graph/badge.svg?token=9AIEBTKLCC)](https://codecov.io/gh/edgexfoundry/device-onvif-camera) [![Go Report Card](https://goreportcard.com/badge/github.com/edgexfoundry/device-onvif-camera)](https://goreportcard.com/report/github.com/edgexfoundry/device-onvif-camera) [![GitHub Latest Dev Tag)](https://img.shields.io/github/v/tag/edgexfoundry/device-onvif-camera?include_prereleases&sort=semver&label=latest-dev)](https://github.com/edgexfoundry/device-onvif-camera/tags) ![GitHub Latest Stable Tag)](https://img.shields.io/github/v/tag/edgexfoundry/device-onvif-camera?sort=semver&label=latest-stable) [![GitHub License](https://img.shields.io/github/license/edgexfoundry/device-onvif-camera)](https://choosealicense.com/licenses/apache-2.0/) ![GitHub go.mod Go version](https://img.shields.io/github/go-mod/go-version/edgexfoundry/device-onvif-camera) [![GitHub Pull Requests](https://img.shields.io/github/issues-pr-raw/edgexfoundry/device-onvif-camera)](https://github.com/edgexfoundry/device-onvif-camera/pulls) [![GitHub Contributors](https://img.shields.io/github/contributors/edgexfoundry/device-onvif-camera)](https://github.com/edgexfoundry/device-onvif-camera/contributors) [![GitHub Committers](https://img.shields.io/badge/team-committers-green)](https://github.com/orgs/edgexfoundry/teams/device-onvif-camera-committers/members) [![GitHub Commit Activity](https://img.shields.io/github/commit-activity/m/edgexfoundry/device-onvif-camera)](https://github.com/edgexfoundry/device-onvif-camera/commits)
 
-
 > **Warning**  
-> The `main` branch contains work-in-progress development code for the upcoming EdgeX 3.0.0 release, **and is not guaranteed to be stable or working**. It is only compatible with the [main branch of edgex-compose](https://github.com/edgexfoundry/edgex-compose).
+> The **main** branch of this repository contains work-in-progress development code for the upcoming release, and is **not guaranteed to be stable or working**.
+> It is only compatible with the [main branch of edgex-compose](https://github.com/edgexfoundry/edgex-compose) which uses the Docker images built from the **main** branch of this repo and other repos.
 >
-> **The current stable branch of device-onvif-camera is [Levski](https://github.com/edgexfoundry/device-onvif-camera/tree/levski).**
+> **The source for the latest release can be found at [Releases](https://github.com/edgexfoundry/device-onvif-camera/releases).**
 
 ### General
 Visit the [EdgeX documentation page](https://docs.edgexfoundry.org/3.0/microservices/device/supported/device-onvif-camera/General/) to learn more about the latest release.
```

Comment:
```suggestion
Visit the [EdgeX documentation page](https://docs.edgexfoundry.org/latest/microservices/device/supported/device-onvif-camera/General/) to learn more about the latest release.
```
This is related to the comments left on the usb [remove-docs pr](https://github.com/edgexfoundry/device-usb-camera/pull/238), but I think this is relevant to this pr.

Reply:
updated.